kernel/kernel_config-4.9.x

```diff
@@ -243,16 +243,13 @@ CONFIG_OPROFILE=y
 # CONFIG_OPROFILE_EVENT_MULTIPLEX is not set
 CONFIG_HAVE_OPROFILE=y
 CONFIG_OPROFILE_NMI_TIMER=y
-CONFIG_KPROBES=y
+# CONFIG_KPROBES is not set
```

Comment:
no let's leave it. `kprobes` are quite invasive. Events are nice and more important to have for `perf`. Do you know if it blocks the eBPF style hooks for monitoring or is this done via a different flags as well?